affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleDetector.kt

```diff
@@ -113,8 +111,39 @@ abstract class AffectedModuleDetector {
         private const val ENABLE_ARG = "affected_module_detector.enable"
         var isConfigured = false
 
+        private var affectedModuleConfiguration: AffectedModuleConfiguration? = null
+
+        @JvmStatic
+        fun getRootConfiguration(project: Project): AffectedModuleConfiguration {
+            if (affectedModuleConfiguration == null) {
```

Comment:
rootConfiguration getting was called every time when "shouldInclude" fun was called. Hence, I transformed it to the singletone.

Reply:
As I posted in the other channel, getting the configuration is a map so there is no runtime benefit to this